### ISSUE TITLE
Debian binary packages page should specify that only AMD64 architecture is currently supported.

### ISSUE THREAD

**Issue by ArloJamesBarnes** [zcash/zcash#1994]
As near to the beginning of the page as possible. I686 user here :(

**Comment by arcalinea** [comment]
Hi @ArloJamesBarnes , the [Debian binary packages ](https://github.com/zcash/zcash/wiki/Debian-binary-packages) wiki page says that our package repository is for 64-bit Debian based distributions--sorry if that wasn't clear. I'll add an additional note. Where in the installation process did you realize your architecture was incompatible?

**Comment by bitcartel** [comment]
Closing as page does specify x86-64 architecture.

**Comment by ArloJamesBarnes** [comment]
On Mon, Jan 9, 2017 at 10:37 AM, Jay Graber <notifications@github.com>
wrote:

> Where in the installation process did you realize your architecture was
> incompatible?


Sorry for the long delay in replying to this. When I ran the sudo apt-get
update && sudo apt-get install zcash I got (after the update stuff) the
error

Reading state information... Done
E: Unable to locate package zcash

So I went to https://apt.z.cash to see what was up, and saw
<https://apt.z.cash/pool/main/z/zcash/> the architecture in the names. To
confirm, ran sudo dpkg -i zcash-1.0.4-amd64.deb and got
dpkg: error processing archive zcash-1.0.4-amd64.deb (--install):
 package architecture (amd64) does not match system (i386)
Errors were encountered while processing:
 zcash-1.0.4-amd64.deb

I now think it was my fault for not paying better attention, so closing the
issue was the right move. I guess I was just annoyed at the time.

Best of luck as the project progresses!
Thanks,
—Arlo James Barnes
